sidekick/sidekick-broker/sidekick-broker-app/src/main/resources/application.yml

```diff
@@ -28,4 +28,10 @@ broker:
     authToken: test
 
 amq:
-  webhookQueueName: webhook-queue
\ No newline at end of file
+  webhookQueueName: webhook-queue
+
+app:
+  version: @project.version@
+
+phone-home:
+    url: http://127.0.0.1:3000/phone-home
```

Comment:
Doesn't it send to our backend?

Reply:
It will be send to our new lambda. I will update the url when lambda deployed to production.